omnibus/config/software/chef.rb

```diff
@@ -69,13 +69,6 @@
   block { log.info(log_key) { "" } }
   bundle "install --verbose", env: project_env
 
-  # For whatever reason, nokogiri software def deletes this (rather small) directory
```

Comment:
we may have stopped shipping dev gems in the build the way we used to?  i dunno..  as long as you test through manhattan.ci on solaris + aix its fine...